controllers/rhmi/rhmi_controller.go

```diff
@@ -174,6 +186,9 @@ func New(mgr ctrl.Manager) *RHMIReconciler {
 // Permission to get the ConfigMap that embeds the CSV for an InstallPlan
 // +kubebuilder:rbac:groups="",resources=configmaps,verbs=get
 
+// Permission to get the serviceaccount
+// +kubebuilder:rbac:groups="",resources=serviceaccounts,verbs=get;list;watch
+
```

Comment:
Do we still need to add this permission? Now that you use bearer token from restConfig...

Reply:
Good spot, No we don't. I've removed it now.